docs/pages/button.js

```diff
@@ -259,245 +229,481 @@ card(
+            onDismiss={() => {}}
+            positionRelativeToAnchor={false}
+            size={240}
+          >
+            <Box padding={3}>
+              <Flex alignItems="center" direction="column" gap={4}>
+                <Text color="white" align="center">
+                  New look! Click Created to see Pins you've published. Click Saved to see your saved Pins and boards.
+                </Text>
+                <Flex alignItems="stretch" direction="column" gap={2}>
+                  <Button
+                    color="white"
+                    size="lg"
+                    text="Got it"
+                    fullWidth
+                  />
+                  <Button
+                    color="semiTransparentWhite"
+                    size="lg"
+                    text="Learn more"
+                    fullWidth
+                  />
+                </Flex>
+              </Flex>
+            </Box>
+          </Popover>
+        </Layer>}
+    </ScrollBoundaryContainer>
+  )
+}
 `}
-  />,
-);
-
-card(
-  <Example
-    name="Selected state"
-    id="selected"
-    defaultCode={`
-function Example() {
-  const [selected, setSelected] = React.useState(false);
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="Icons"
+      description={`
+1. Icon end
+Adds an icon after the Button text. Icons should only be used to reinforce a specific function or interaction of the Button. Menus and external links are a common use case. Use \`arrow-up-right\` when linking to an external URL or \`arrow-down\` when displaying a Popover on click. Note that iconEnd on Button is not accessible to screen readers.
+`}
+    >
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Button
+  accessibilityLabel='Menu'
+  iconEnd="arrow-down"
+  size="lg"
+  text="Menu"
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      columns={2}
+      title="Role"
+      description={`
+1. Button (default)
+    The “button” \`role\` is used for actions. This is the default and should be used for most Buttons.
+2. Link
+    The “link” \`role\` is used for navigating by URL. These Buttons should not use a \`selected\` state.
+`}
+    >
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Button
+  accessibilityLabel='Follow'
+  size="lg"
+  text="Follow"
+  role="button"
+/>
+`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        defaultCode={`
+<Button
+  accessibilityLabel='Visit Pinterest'
+  iconEnd="arrow-up-right"
+  size="lg"
+  text="Visit Pinterest"
+  role="link"
+  href="https://pinterest.com"
+/>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      title="rel and target"
+      description={`
+These optional props control the behavior of \`role="link"\` Buttons. External links commonly use \`target="_blank"\` to open the link in a new tab or window, and \`rel="nofollow"\` to provide hints for SEO.
```

Comment:
![image](https://user-images.githubusercontent.com/10593890/132907014-7c40435f-e5d4-4ea3-bdc6-c927c6457eca.png)


Could we fix the typos in `IconButton`?

rel is optional. Use "**nofollow**" for external links to specify to web crawlers not **to** follow the link.

Reply:
Not sure if I follow - is this a change you'd like to make to IconButton?

Reply:
@AlbertCarreras The IconButton fix should be in a separate PR: https://github.com/pinterest/gestalt/pull/1691